clustered/common/src/main/java/org/ehcache/clustered/common/internal/messages/ServerStoreOpMessage.java

```diff
@@ -23,12 +23,12 @@
 public abstract class ServerStoreOpMessage extends EhcacheEntityMessage implements ConcurrentEntityMessage {
   public enum ServerStoreOp {
 
-    GET((byte) 10),
-    GET_AND_APPEND((byte) 11),
-    APPEND((byte) 12),
-    REPLACE((byte) 13),
-    CLIENT_INVALIDATION_ACK((byte) 14),
-    CLEAR((byte) 15),
+    GET((byte) 11),
+    GET_AND_APPEND((byte) 12),
+    APPEND((byte) 13),
+    REPLACE((byte) 14),
+    CLIENT_INVALIDATION_ACK((byte) 15),
+    CLEAR((byte) 16),
```

Comment:
Why renumbering the op codes? This is okay as we said we wouldn't guarantee compatibility across versions for now but I fail to see the benefit here.